exec/java-exec/src/main/resources/rest/query/query.ftl

```diff
@@ -82,6 +82,16 @@
       Submit
     </button>
     <input type="checkbox" name="forceLimit" value="limit" <#if model.isAutoLimitEnabled()>checked</#if>> Limit results to <input type="text" id="autoLimit" name="autoLimit" min="0" value="${model.getDefaultRowsAutoLimited()?c}" size="6" pattern="[0-9]*"> rows <span class="glyphicon glyphicon-info-sign" title="Limits the number of records retrieved in the query. Ignored if query has a limit already" style="cursor:pointer"></span>
+    <label> Store Format
+      <select name="store.format">
+        <option value="parquet">Parquet</option>
+        <option value="json">JSON</option>
+        <option value="csv">Comma-Separated Values</option>
+        <option value="psv">Pipe-Separated Values</option>
+        <option value="tsv">Tab-Separated Values</option>
```

Comment:
@dobesv As I have stated before, we should not hard-code format names since they can be customized, please remove this part of code, other changes are ok.

Reply:
@arina-ielchiieva OK, I'll just remove the field for now.  I added it here for demo/testing but I don't actually need it to be there for my own purposes.